mlrun/projects/project.py

```diff
@@ -1517,6 +1517,7 @@ def get_artifact(spec):
                     artifact, local_path=temp_dir, artifact_path=artifact_path
                 )
 
+            # TODO - unreachable (weak) code (see return)
```

Comment:
looks like a mistake. @yaronha can confirm butregardless, can be removed (and not TODO'ed)

Reply:
@liranbg,  I hoped, that note will be remove this part of code or move it before return ... let me know.

Reply:
look like line 1520 should move up, before the return and after 1503 (i.e. delete the local copy of the zip file when the data source is a remote object (not file))